source/sphinx_extensions/plugin_directory/templates/plugin.rst

```diff
@@ -6,6 +6,10 @@
    <table class="table plugin-info">
      <tbody>
        <tr>
+         <th scope="row">Description</th>
+         <td>{{ plugin.description|urlize }}</td>
```

Comment:
Can you take the same approach as the `Support` and `Citation` sections below? Since these are free text blocks, the text has to be split into lines, where each line is `urlized` individually (I tried it without breaking lines and `urlize` doesn't work as expected).